### DISCUSSION TITLE
连接、沟通与信任 —— 开源项目需要什么样的用户共同体以及如何构建？ - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#20] [Announcements]
# 连接、沟通与信任 —— 开源项目需要什么样的用户共同体以及如何构建？ - 开源之道

https://opensourceway.community/posts/community_leadership_development/what-user-community-we-need/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

harley-davidson 是个很好的类比, 但是, 完全没有展开哪,
harley-davidson 和我们嘉陵机车用户间有什么不同?

是什么塑造出了 harley-davidson ? 厂商和用户间有什么样的固定交流?
为什么有的人选择了 harley-davidson 生活方式? 
...

一系列问题如果不深入阐述, 直接来宣布 开源用户共同体, 是完全没有感召力的:

- 毕竟 harley-davidson 在现实社会中也是另类, 几乎是暴力团伙的最佳形象代言人, 可人家只是喜欢在高速公路上飙机车而已
- 而且 harley-davidson 生活成本不低, 毕竟机车本身再贵, 贵不到哪儿去, 真正给厂商续命的, 是各种周边产品, 以及保养工具的销售
- 是的, 和开源作品用户一样, harley-davidson 必须能独立完备保养所有零件, 紧急情况时, 甚至于必须知道如何自制/临时用其它品牌零件来完成恢复工作, 这和开源作品必须能独立定制, 才可能良好嵌入到自己工作场景中非常类似

也就是说, 没有所谓 开源作品用户共同体, 只有开源共同体, 要知道 Hacker 有五大类:

+ 开发 ~ developer
+ 测试 ~ tester
+ 文档 ~ documenter
+ 运营 ~ runner
+ 宣传 ~ Evangelist 

而且不分高下, 多数开源项目创始人, 同时要作这五个方向上的事儿, 直到社区发展到可以分工, 才专注核心作品的研发...

所以, 一名合格的开源用户, 必然贡献了以上五个方向上的一些知识, 本质上不是单纯的最终用户了, 已经是开源社区成员了...
